concent-vm/helper-scripts/golem-run-gui-mode.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash -e
+
+trap "kill 0" SIGINT
+
+golem_run_gui_command="$(
+    golem-run-console-mode.sh $@ &    \
+    cd ~/golem_electron/;             \
+    npm run start:app &               \
+    npm run start:mainnet &           \
+)"
+
+
+$golem_run_gui_command
```

Comment:
Why put the command in a variable? You can just execute it.

Reply:
Turns out "calling" the variable is not necessary but the purpose of `$()` is that it allows `trap` to kill all child processes on Ctrl+C. I've cleaned it up a bit but kept it more or less the same.